resources/js/components/FileCard.vue

```diff
@@ -26,7 +26,18 @@
         >
             <button
                 type="button"
-                v-if="withMeta.canDownload"
+                @keydown.enter.prevent="viewFile"
+                @click.prevent="viewFile"
+                v-tooltip.click="__('View')"
+                v-if="withMeta.canDownload && withMeta.contentDisposition !== 'download'"
+                class="cursor-pointer dim btn btn-link text-primary inline-flex items-center ml-3"
+            >
+                <icon type="view" view-box="0 0 24 24" width="16" height="16" />
```

Comment:
Better use this icon https://github.com/tailwindlabs/heroicons/blob/master/optimized/outline/external-link.svg and keep the eye/view icon for viewing the model resource detail page

Reply:
I'm not sure how put this icon,

I used svg

Reply:
Try this
```
<svg
    xmlns="http://www.w3.org/2000/svg"
    class="fill-current"
    width="16"
    height="16"
    viewBox="0 0 24 24"
    aria-labelledby="link"
    role="presentation"
  >
    <path d="M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14"/>
</svg>
```

Reply:
I used it !!!